tests/i18n/test_extraction.py

```diff
@@ -204,6 +204,16 @@ def test_comments_extractor(self):
                 po_contents
             )
 
+    def test_location_comments_for_templatized_files(self):
+        os.chdir(self.test_dir)
+        management.call_command('makemessages', locale=[LOCALE], verbosity=0)
+        self.assertTrue(os.path.exists(self.PO_FILE))
+        with io.open(self.PO_FILE, 'r', encoding='utf-8') as fp:
+            po_contents = fp.read()
+            pattern = '^#:.+\.html\.py.*$'
+            matches = re.findall(pattern, po_contents, flags=re.MULTILINE)
+            self.assertFalse(matches, 'Found an unprocessed file name in comment line')
```

Comment:
Can we make any positive assertion? I'm a bit nervous about a negative assertion like this since, for example, a typo in the regular expression could cause the test to pass by mistake.